src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -921,4 +922,63 @@ public String getMappedColumnName() {
         }
     }
 
+    /**
+     * This method is used to validate the dates of an exercise
```

Comment:
I think the documentation could be extended by explaining more what exactly this method is doing. 
Could you add more details that shows the real logic behind this code, e.g. release date before due date etc.